tests/gtest/test_population_dynamics_fleet.cpp

```diff
@@ -0,0 +1,45 @@
+#include "gtest/gtest.h"
+#include "population_dynamics/fleet/fleet.hpp"
+#include "distributions/distributions.hpp"
+
+namespace
+{
+
+  //this demostrates how to set up to test the structure of fleets.
+  // It won't work in gtest because the Dlnorm functions are only available if
+  // TMB is also available. This test instead should be done as a testthat rcpp
+  // test rather than gtest. However, we are leaving this file to provide an example
+  // of the approach.
+  TEST(fleet, create_object)
+  {
+    //set up selectivity
+    fims::LogisticSelectivity<double> fishery_selectivity;
+    fishery_selectivity.median = 20.5;
+    fishery_selectivity.slope = 0.2;
+    double fishery_x = 40.5;
+    // 1.0/(1.0+exp(-(40.5-20.5)*0.2)) = 0.9820138
+    double expect_fishery = 0.9820138;
+
+    //set up index likelihood
+    fims::Dlnorm<T> nll_dlnorm;
+    nll_dlnorm.logy = 2.356247;
```

Comment:
May be useful to indicate where the calculation of nll_dlnorm.logy  = 2.356247
comes from

Reply:
@Andrea-Havron-NOAA or @Bai-Li-NOAA do you know where this comes from?

Reply:
@k-doering-NOAA, I do not know where this value comes from but I have a suggestion. My bad that I asked Andrea to see if we can set up a fleet C++ test and forgot to discuss the next steps when finding out the test did not work. The current c++ test won't work in gtest because the Dlnorm functions are only available if TMB is also available. The current c++ test does not include any actual tests to compare true values and FIMS objects. Therefore, I suggest  we remove the test_population_dynamics_fleet.cpp from the branch. This test instead should be done as a [testthat rcpp test](https://github.com/NOAA-FIMS/FIMS/blob/179-feat-fleet-module/tests/testthat/test-fleet-interface.R) rather than gtest.

Reply:
@k-doering-NOAA removed the test_population_dynamics_fleet.cpp file so we are resolving!